### DISCUSSION TITLE
MP3 link for new songs

### DISCUSSION THREAD

**Discussion by TharinduWishwadeepa** [cyberboysumanjay/JioSaavnAPI#42] [Q&A] [answered]
Can't we get mp3 320 kbps links for songs using this link
https://apg-saavn-api.herokuapp.com/result/?q=

**Comment by cyberboysumanjay** [comment, marked as answer]
We can generate links for mp3 also using a simple string manipulation on top of the current url.
Read the details and how to do here in this issue https://github.com/cyberboysumanjay/JioSaavnAPI/issues/33#issuecomment-809903245

The problem with this approach is now they're not adding mp3 for all the songs so API might break in case I force it to serve mp3 which is not a good idea so I've kept it to serve m4a instead of mp3 which actually makes sense because it's approximately smaller in size and better in quality than mp3.

**Reply by TharinduWishwadeepa** [reply]
Thank you for the answer

**Comment by tanojsaipraveen** [comment]
Nice api ,its a great work 
how to decrypt the urls and how did you find that decryption key is there any to find it
